templates/zerver/help/alert-words.md

```diff
@@ -1,28 +1,34 @@
-# Alert Words
-Zulip allows you to define custom alert words and phrases.
-When they are used in any stream you are subscribed to, it works
-similarly to being [@-mentioned](/help/at-mention-a-team-member)
-by another user.
+# Alert words
+
+Zulip allows you to define custom alert words and phrases. When they are used
+in any stream you are subscribed to, it works similarly to being
+[@-mentioned](/help/at-mention-a-team-member) by another user.
+
+## Adding alert words
 
-## Adding Alert Words
 {!go-to-the.md!} [Custom Alert Words](/#settings/custom-alert-words)
 {!settings.md!}
 
-2.  Type the word/phrase you wish to be notified for into the text box,
-    then click the **Add Alert Word** button.
+2. Type the word/phrase you wish to be notified for into the **New Alert Word**
+text box.
```

Comment:
Maybe add `e.g. Downtime`? It took me a few seconds to notice the alert word in the screenshot, didn't know what I was looking for.